src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class Machine {
+    public void doWork() {
+    }
+
+    public void stopWork() {
+    }
```

Comment:
the whole point of abstract classes is that they do not have an implementation/realization/body, just the general idea of what they are supposed to do
```suggestion
    public abstract void doWork();

    public abstract void stopWork();
```